connector.go

```diff
@@ -50,20 +50,25 @@ import (
 //  * https://docs.aws.amazon.com/AWSEC2/latest/APIReference/errors-overview.html#CommonErrors
 //  * https://docs.aws.amazon.com/STS/latest/APIReference/CommonErrors.html
 func NewAWSReader(
-	ctx context.Context, accessKey string, secretKey string, regions []string, config *aws.Config,
-) (AWSReader, error) {
+	ctx context.Context, accessKey string, secretKey string, regions []string, config *aws.Config, customEndpoint bool) (AWSReader, error) {
```

Comment:
So goo docs but I want them also on the function description of `NewAWSReader` hehe